src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
```

Comment:
**Issue**: The condition in the loop has been changed to compare the numbers directly instead of their absolute values. This modification will cause the method to return the maximum value, not the absolute maximum as intended. <br> **Fix**: Restore the condition to compare the absolute values of the numbers to ensure that the method returns the absolute maximum value. <br> **Code Suggestion**: 
 ```
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```